spynnaker8/utilities/spynnaker8_neo_segment.py

```diff
@@ -1,6 +1,13 @@
-from neo import Segment, SpikeTrain, AnalogSignalArray
+from neo import Segment, SpikeTrain
 import numpy
 import quantities as pq
+# Or neo version
+from spynnaker8.utilities.version_util import pynn8_syntax as pynn8_syntax
+if pynn8_syntax:
+    from neo import AnalogSignalArray as AnalogSignal
+else:
+    from neo import AnalogSignal as AnalogSignal
```

Comment:
Another…

Reply:
I am leaving this one for now as it highlights the compatibility workaround.
Different classes imported with same name.

Agree that when pynn8 removed the as should be too.